share/p2p/peers/options.go

```diff
@@ -82,3 +82,10 @@ func (m *Manager) WithMetrics() error {
 	m.metrics = metrics
 	return nil
 }
+
+func WithTag(tag string) func(manager *Manager) error {
+	return func(m *Manager) error {
+		m.tag = tag
+		return nil
+	}
+}
```

Comment:
Should be part of the constructor, because it is not optional - needs to be set explicitly.

Reply:
https://github.com/celestiaorg/celestia-node/pull/3188/commits/b48d69673b41689be9a2ff4012c9e4a20c35649c